src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -159,6 +159,7 @@
       <mat-card-subtitle class="m-0">{{ selectedVersion?.name }}</mat-card-subtitle>
     </mat-card-header>
     <mat-card-content class="p-3">
+      <div><strong matTooltip="Digital Object Identifier">DOI</strong>: {{ selectedVersion.doiURL ? selectedVersion.doiURL : 'n/a' }}</div>
```

Comment:
selectedVersion can be null apparently, based on line159, so `{{ selectedVersion && selectedVersion.doiURL...`. Not sure how the Angular question mark works with the ternary operator, but there could be a more elegant way as well.

Reply:
done